impl_core/src/test/java/io/opencensus/implcore/stats/StatsRecorderImplTest.java

```diff
@@ -110,6 +114,37 @@ public void record_CurrentContextSet() {
     assertThat(viewData.getAggregationMap().keySet()).containsExactly(Arrays.asList(VALUE));
   }
 
+  @Test
+  public void record_UnregisteredMeasure() {
+    View view =
+        View.create(
+            VIEW_NAME,
+            "description",
+            MEASURE_DOUBLE,
+            Sum.create(),
+            Arrays.asList(KEY),
+            Cumulative.create());
+    viewManager.registerView(view);
+    Context orig =
+        Context.current()
+            .withValue(ContextUtils.TAG_CONTEXT_KEY, new SimpleTagContext(Tag.create(KEY, VALUE)))
+            .attach();
+    try {
+      statsRecorder
+          .newMeasureMap()
+          .put(MEASURE_DOUBLE_NO_VIEW_1, 1.0)
```

Comment:
nit: might be useful to have 3 different values recorded, and verify that the value associated with MEASURE_DOUBLE is in `viewData`.

Reply:
Done.